bundle/manifests/netobserv-operator.clusterserviceversion.yaml

```diff
@@ -469,7 +469,7 @@ spec:
       name: flowcollectors.flows.netobserv.io
       specDescriptors:
       - description: defines the desired type of deployment for flow processing.
-        displayName: Deployment model
+        displayName: udeployment LModel
```

Comment:
not sure how you did that, but all those names look pretty funky !

Reply:
yeah I just reverted the changes to that file